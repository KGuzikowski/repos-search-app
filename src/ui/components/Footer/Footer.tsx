import React from 'react'
import styled from 'styled-components'
import { primaryColor, secondaryColor } from '../../style-variables'

const FooterDiv = styled.div`
    width: 100%;
    box-sizing: border-box;
    padding: 30px 20px;
    font-size: 20px;
    text-align: center;
    background: ${primaryColor};
    color: #fff;

    @media only screen and (max-width: 700px) {
        font-size: 16px;
        padding: 20px 10px;
    }
`

const Name = styled.span`
    color: ${secondaryColor};
`

const Footer = () => {
    return (
        <FooterDiv>
            Let's search for repositories! App made for recruitment purposes by <Name>Karol Guzikowski</Name>.
        </FooterDiv>
    )
}

export default Footer