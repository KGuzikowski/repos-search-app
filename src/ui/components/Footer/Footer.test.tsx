import React from 'react'
import { create } from 'react-test-renderer'
import Footer from './Footer'

it('renders Footer component correctly', () => {
    const component = create(<Footer />)
    expect(component.toJSON()).toMatchSnapshot()
})