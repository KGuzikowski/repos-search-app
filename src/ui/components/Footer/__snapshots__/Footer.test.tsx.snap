// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer component correctly 1`] = `
<div
  className="sc-AxjAm fLHnvz"
>
  For Callstack team! App made for recruitment purposes by 
  <span
    className="sc-AxirZ eBjaA-D"
  >
    Karol Guzikowski
  </span>
  .
</div>
`;
